tests/ssh-hassh/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 5.0.0
+features:
+    - RUST
+
+args:
+ - -k none
+
+checks:
+  # Check that we have the following events in eve.json
+  # 2 - alert events
+  # 1 - tls event
+  - filter:
+      count: 4
+      match:
+        event_type: alert
+  - filter:
+      count: 1
+      match:
+        event_type: ssh
```

Comment:
Should we add a deeper filter to check the value of ssh.hassh ?

Reply:
it's checked already in rules (rule will match only for specific hassh values).

Reply:
Ok, so should we remove the whole filter of one `event_type == ssh` ?
Or is there more to it ?

Reply:
yes, I think, it can be removed.